config/jobs/etcd/etcd-operator-postsubmits.yaml

```diff
@@ -47,8 +47,8 @@ postsubmits:
         - bash
         - -c
         - |
+          # We can't remove the line below for now, because the e2e test needs the kind binary present
```

Comment:
May be a dumb question, but why do they still need the binary?

Reply:
Essentially, the e2e framework just executes the binary to create the kind cluster, 
https://github.com/etcd-io/etcd-operator/blob/f03e687902544f472ab3a48e3e6c0e491f3da82a/test/e2e/e2e_suite_test.go#L56

Reply:
Interesting, I thought it had the binary vendored into the package.